docker/ol-web-start.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+if [[ "$USE_NGINX" ]]; then
+  mkdir /etc/nginx || true
+  rm -f /etc/nginx/nginx.conf || true
+  ln --verbose --symbolic /opt/olsystem/etc/nginx/nginx.conf      /etc/nginx/nginx.conf
+  ln --verbose --symbolic /opt/olsystem/etc/nginx/sites-available /etc/nginx/sites-available
```

Comment:
`sudo` is a command not found inside the Docker container so how do I get around these errors?
```
web_1        | mkdir: cannot create directory '/etc/nginx': File exists
web_1        | rm: cannot remove '/etc/nginx/nginx.conf': Permission denied
web_1        | ln: failed to create symbolic link '/etc/nginx/nginx.conf': File exists
web_1        | ln: failed to create symbolic link '/etc/nginx/sites-available/sites-available': Permission denied
```

Reply:
`ln -sf`